Task1/Program.cs

```diff
@@ -10,7 +10,152 @@ class Program
     {
         static void Main(string[] args)
         {
-            
+            //DeleteDublicate(EnterString());
+            //NumberOfSymbol(EnterString());
+            //InsertString(EnterString(), 7);
+            //DeleteSpace(EnterString());
+            //PrintFirstLetter(EnterString());
+            //PrintReverseOrder(EnterString());
+            //LengthOfWord(EnterString());
+            DeleteExtraSpace(EnterString());
+
+            Console.ReadKey();
+        }
+
+        public static string EnterString()
+        {
+            Console.WriteLine("Enter string:");
+            return Console.ReadLine();
+        }
+
+        public static void DeleteDublicate(string str)
+        {
+            string res = string.Empty;
+            bool isDublicate;
+
+            for (int i = 0; i < str.Length; i++)
+            {
+                isDublicate = false;
+                for (int j = 0; j < res.Length; j++)
+                {
+                    if (str[i] == res[j])
+                    {
+                        isDublicate = true;
+                    }
+                }
+                if(isDublicate == false)
```

Comment:
It is not needed to compare bool with bool to get bool result.
Please use bool value in if statement:
if(!isDuplicate)